eventing/samples/kubernetes-event-source/README.md

```diff
@@ -19,7 +19,7 @@ consumption by a function that has been implemented as a Knative Service.
    later.
 
 ```shell
-kubectl -n default apply -f channel.yaml
+kubectl --namespace  default apply --filename channel.yaml
```

Comment:
There are extra spaces being added after namespace in several places throughout this PR. Can you remove them? @gyliu513

Reply:
Done, thanks @samodell